tests/tests_suite_test.go

```diff
@@ -140,3 +141,25 @@ var _ = ReportAfterSuite("TestTests", func(report Report) {
 var _ = JustAfterEach(func() {
 	k8sReporter.ReportSpec(CurrentSpecReport())
 })
+
+func testCleanup() {
```

Comment:
To me, it would be easier to review this change in its own PR.

Reply:
This does not look a huge change IMHO. The commit is very easy to read and affect only 2 files, with few line changes. Am I missing smth? Thanks